libraries/native/native/eosio/crt.hpp

```diff
@@ -1,19 +1,25 @@
 #pragma once
 #include <setjmp.h>
 
+#include <string>
+
 namespace eosio { namespace cdt {
    enum output_stream_kind {
       std_out,
       std_err,
       none
    };
-   struct output_stream {
-      char output[1024*2];
-      size_t index = 0;
-      std::string to_string()const { return std::string((const char*)output, index); }
-      const char* get()const { return output; }
-      void push(char c) { output[index++] = c; }
-      void clear() { index = 0; }
+   class output_stream {
+      static constexpr size_t initial_size = 1024 * 4;
+      std::string output;
+
+      public:
+      output_stream() { output.reserve(initial_size); }
+      std::string to_string() const { return output; }
```

Comment:
Right, after we talked yesterday I have changed it, so now it is const ref.
I believe originally it was a copy, because the underlying was a char array.